django_project/geosight/data/tests/api/context_layer.py

```diff
@@ -143,3 +145,37 @@ def test_delete_multiple_api(self):
             url, 200, self.creator, data={'ids': json.dumps([resource.id])}
         )
         self.assertEqual(ContextLayer.objects.count(), 1)
+
+
+class TestRasterZonalAnalysis(BasePermissionTest.TestCase):
+
+    def setUp(self):
+        self.context_layer = ContextLayerF(
```

Comment:
This is not a public tif
Maybe find other small cog example and download it and put it as static file and use the static url for test

Reply:
I've updated it to use mock